test/drbgtest.c

```diff
@@ -423,7 +427,7 @@ static int error_check(DRBG_SELFTEST_DATA *td)
     if (!instantiate(drbg, td, &t))
         goto err;
     reseed_counter_tmp = drbg->reseed_gen_counter;
-    drbg->reseed_gen_counter = drbg->reseed_interval;
+    drbg->reseed_gen_counter = drbg->reseed_interval + 1;
```

Comment:
This no longer works. Since https://github.com/openssl/openssl/commit/8bf3665#diff-9181ac017a6177a5f2619f65c9b7a346R682, the counter must be larger than the interval to trigger the reseeding.

Reply:
I'll need to check that tomorrow...

Reply:
IIRC this was a deliberate change by @slontis because in the NIST document the counter counts from 1 and not from 0. (see https://github.com/openssl/openssl/commit/8bf3665#diff-9181ac017a6177a5f2619f65c9b7a346R516.) Is that correct @slontis?